pandas/io/parsers.py

```diff
@@ -91,7 +91,9 @@
 prefix : str, default None
     Prefix to add to column numbers when no header, e.g. 'X' for X0, X1, ...
 mangle_dupe_cols : boolean, default True
-    Duplicate columns will be specified as 'X.0'...'X.N', rather than 'X'...'X'
+    Duplicate columns will be specified as 'X.0'...'X.N', rather than
+    'X'...'X'. Passing in False will cause data to be overwritten if there
```

Comment:
maybe add something to the ``names`` part of the doc-string as well (e.g. saying if you have dupes then pay attention to mangle_dupe_cols)